src/Websocket.Client/WebsocketClient.cs

```diff
@@ -20,8 +21,12 @@ namespace Websocket.Client
     /// </summary>
     public partial class WebsocketClient : IWebsocketClient
     {
-        private static readonly ILog Logger = GetLogger();
+        /// <summary>
+        /// Logger factory used in case no implementation is passed to ctor
+        /// </summary>
+        public static ILoggerFactory LoggerFactory { get; set; } = new NullLoggerFactory();
```

Comment:
I would make it **private** with 'new NullLoggerFactory' default value, as you have. If someone wants to use the log, he will pass it into the ctor, if not, he won't and the NullLogger will be used, clear and simple. With your approach, it is not clear what is the best way to pass the logger in, either via the ctor or via this static LoggerFactory property with NULL for the ctor.

Reply:
Changed to private static readonly